src/commodities/AbstractCommodity.java

```diff
@@ -17,6 +17,32 @@ public void setInventoryNumber(int inventoryNumber) {
         this.inventoryNumber = inventoryNumber;
     }
 
+    @Override
```

Comment:
By convention the source files should be in src/main/java and the test classes in src/test/java.
Also lets create a package **hotel.service.domain** in which we can add the **commodities** package